apps/voting/contracts/Voting.sol

```diff
@@ -199,7 +199,7 @@ contract Voting is IForwarder, AragonApp {
         }
 
         // Voting is already decided
-        if (_isValuePct(vote_.yea, vote_.totalVoters, vote_.supportRequiredPct)) {
+        if (_isValuePct(vote_.yea, vote_.totalVoters, uint256(vote_.supportRequiredPct))) {
```

Comment:
I know @izqui commented on this in https://github.com/aragon/aragon-apps/pull/473/files#r224013414, but across function calls the cast is redundant as the solidity compiler automatically casts it for us.

We never actually do any math involving `minAcceptQuorumPct` and `supportRequiredPct` outside of a comparison (which would also automatically be casted for us), and it should be a red flag if we ever did.

I wouldn't mind removing the explicit upwards casts @izqui.